### ISSUE TITLE
LSB linker

### ISSUE THREAD

**Issue by nedrichards** [flatpak/freedesktop-sdk-images#56]
I was making an extra-data based flatpak for Google Earth but it needs the LSB to run, which isn't in the FDo runtime. I attempted to softlink the location of the linker:

> Running: ln -s /lib/ld-linux-x86-64.so.2 /lib/ld-lsb-x86-64.so.3

But that obviously doesn't work since apps shouldn't be doing that. Could we look at adding some symlinks to support this?

**Comment by alexlarsson** [comment]
Makes sense to me, and should be easy to add. Anyone want to take a shot?

**Comment by matwey** [comment]
Any news? It seems that I've hit the same issue with another app.
